legacy/builder/phases/linker.go

```diff
@@ -85,7 +85,7 @@ func link(ctx *types.Context, objectFiles paths.PathList, coreDotARelPath *paths
 	properties.Set(constants.BUILD_PROPERTIES_ARCHIVE_FILE_PATH, coreArchiveFilePath.String())
 	properties.Set("object_files", objectFileList)
 
-	_, _, err := builder_utils.ExecRecipe(ctx, properties, constants.RECIPE_C_COMBINE_PATTERN, false /* stdout */, utils.ShowIfVerbose /* stderr */, utils.Show)
+	_, _, err := builder_utils.ExecRecipe(ctx, properties, constants.RECIPE_C_COMBINE_PATTERN, false, utils.ShowIfVerbose /* stdout */, utils.Show /* stderr */)
```

Comment:
Maybe this `false` should also have a comment?

Reply:
The code inside `legacy` requires a general refactoring to make it more idiomatic go and to make it fit in the overall CLI design. I've another PR in the working to do so but turned out very big so I'm trying to make it in smaller pieces, that's why I just fixed the existing comments.

Reply:
Ok!